src/DependencyInjection/Loader/Configurator/RectorContainerConfigurator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+/**
+ * @api
+ * Same as Symfony container configurator, with patched return type for "set()" method for easier DX.
+ * It is an alias for internal class that is prefixed during build, so it's basically for keeping stable public API.
+ */
+final class RectorContainerConfigurator extends ContainerConfigurator
+{
+}
```

Comment:
I could not find the test case, that would actually test this type in `rector.php` results in successfuly loaded Rector container.
Could you one?